eoscostarica/guias.eoscostarica.io
# update(Enlaces utiles): update links

### GH Issue

Resolves #181 

### Steps to test
1. Go to Enlaces Útiles inside of Comunidad tag
1. Check content is right

#### CheckList
